source/Ship.cpp

```diff
@@ -3356,26 +3386,46 @@ double Ship::MinimumHull() const
 
 
 // Find out how much fuel is consumed by the hyperdrive of the given type.
-double Ship::BestFuel(const string &type, const string &subtype, double defaultFuel) const
+double Ship::BestFuel(const string &type, const string &subtype, double defaultFuel, double distance) const
```

Comment:
It's not great to have multiple consecutive parameters of the same type, but I don't know that we can easily change that here. Based on your comment below though, the name should be e.g. `jumpDistance`